NebulaHost/MultiplayerHostSession.cs

```diff
@@ -56,7 +56,7 @@ public void StartServer(int port, bool loadSaveFile = false)
             PacketUtils.RegisterAllPacketNestedTypes(PacketProcessor);
             PacketUtils.RegisterAllPacketProcessorsInCallingAssembly(PacketProcessor);
 
-            socketServer = new WebSocketServer(port);
+            socketServer = new WebSocketServer(System.Net.IPAddress.IPv6Any, port);
```

Comment:
Does this cover both IPv4 and IPv6? Regrettably the world hasn't entirely moved to IPv6 yet

Reply:
Yes it does!